### ISSUE TITLE
Ability Rando Suggestion

### ISSUE THREAD

**Issue by HermitDreams** [Dr4g0nsoul/TunicRandomizer#1]
Maybe locking abilities behind the page that initially teaches you about one would be a good starting point? i.e. you need the page normally inside the Sealed Temple in order to Pray at all

**Comment by Dr4g0nsoul** [comment]
Yeah I already thought about praying and holy cross. Currently my plan is to unlock them from chests/pick-ups.
